SurveyManagementSample/src/main/res/raw/exp_configuration.json

```diff
@@ -0,0 +1,34 @@
+{
+	"customerId":"FSRTESTINGCODECID12345==",
+	"notificationType": "IN_SESSION",
+	"cppParameters":
+	{
+		"foo":"bar"
+	},
+	"invite": {
+		"logo": "exp_logo.png",
+		"header": "invite_header.png",
+		"baseColor": [111, 212, 178]
+	},
+	"survey": {
+		"closeButtonColor": [71, 221, 22],
+		"closeButtonBackgroundColor": [16, 71, 198],
+		"headerColor": [187, 87, 251]
+	},
+	"supportLandscape": true,
+	"surveyManagement": {
+		"surveys":[
+			{
+				"notificationType": "IN_SESSION",
```

Comment:
This instance of `notificationType` should be removed since it's not expected from clients